docs/mobile-sdk.md

```diff
@@ -1,7 +1,940 @@
+</pre>
+
+#### 3️⃣ Updating the font
+
+The font used in the Chatbox can easily be updated using your own font:
+
+##### iOS
+
+<pre class="prettyprint lang-swift">
+var configuration = ChatboxConfiguration()
+configuration.font = UIFont(name: "AmericanTypewriter-Condensed", size: 11.0)
+IAdvizeSDK.shared.chatboxController.setupChatbox(configuration: configuration)
+</pre>
+
+##### Android
+
+<pre class="prettyprint lang-kotlin">
+val configuration = ChatboxConfiguration()
+configuration.fontPath = "fonts/comic_sans_ms_regular.ttf"
+IAdvizeSDK.chatboxController.setupChatbox(configuration)
+</pre>
+
+> *⚠️ The font should be placed inside the assets folder. Here the file is located at `src/main/assets/fonts/comic_sans_ms_regular.ttf`*
+
+#### 4️⃣ Using a brand avatar
+
+The operator avatar displayed alongside his messages can be updated for branding purposes. You can specify a drawable either via an URL or a local resource.
+
+##### iOS
+
+<pre class="prettyprint lang-swift">
+var configuration = ChatboxConfiguration()
+
+// Update the incoming message avatar with a UIImage.
+configuration.incomingMessageAvatar = .image(image: UIImage(named: "BrandAvatar"))
+
+// Update the incoming message avatar with an URL.
+configuration.incomingMessageAvatar = .url(url: "http://avatar.url")
+
+IAdvizeSDK.shared.chatboxController.setupChatbox(configuration: configuration)
+</pre>
+
+##### Android
+
+<pre class="prettyprint lang-kotlin">
+val configuration = ChatboxConfiguration()
+
+// Update the incoming message avatar with a Drawable resource.
+configuration.incomingMessageAvatar =
+  IncomingMessageAvatar.Image(ContextCompat.getDrawable(context, R.drawable.ic_brand_avatar))
+
+// Update the incoming message avatar with an URL.
+configuration.incomingMessageAvatar = IncomingMessageAvatar.Url(URL("http://avatar.url"))
+
+IAdvizeSDK.chatboxController.setupChatbox(configuration)
+</pre>
+
+> *⚠️ GIFs are not supported*
+
+### 🔔 Handling push notifications
+
+> *⚠️ Before starting this part you will need to configure push notifications inside your application. You can refer to the following resources if needed:*
+
+| iOS | Android |
+| --- | ------- |
+| [Push notification setup tutorial](https://www.raywenderlich.com/11395893-push-notifications-tutorial-getting-started) | [Firebase Cloud Messaging documentation](https://firebase.google.com/docs/cloud-messaging/android/client) |
+
+> *You will also need to ensure that the push notifications are setup in your iAdvize project. The process is described above in the [SDK Knowledge Base](https://help.iadvize.com/hc/en-gb/articles/360019839480)*
+
+#### 1️⃣ Registering the device token
+
+For the SDK to be able to send notifications to the visitor’s device, its unique `device push token` must be registered:
+
+##### iOS
+
+<pre class="prettyprint lang-swift">
+IAdvizeSDK.shared.notificationController.registerPushToken("the_device_push_token", applicationMode: .prod)
+</pre>
+
```

Comment:
Tu peux rajouter ça stp : 

You have to pass `.dev` as the `applicationMode` if you want to test the push notifications in `DEBUG`